docs/book/src/developer/tilt.md

```diff
@@ -86,6 +86,8 @@ kustomize_substitutions:
   IBMCLOUD_API_KEY: "XXXXXXXXXXXXXXXXXX"
 ```
 
+> **Note:** If you are using Podman, replace the value of `default_registry` with `localhost:5001` in all occurrences throughout the configuration.
```

Comment:
By default we recommend to use podman so I think we can replace everywhere and set it as localhost:5001.